contracts/test/utils.ts

```diff
@@ -10,7 +10,7 @@ import { Log } from '@ethersproject/providers';
 import { abi as UNISWAP_POOL_ABI } from '@uniswap/v3-core/artifacts/contracts/interfaces/IUniswapV3Pool.sol/IUniswapV3Pool.json';
 import { FeeAmount, Pool, Route, computePoolAddress, encodeRouteToPath } from '@uniswap/v3-sdk';
 import { Token } from '@uniswap/sdk-core';
-import { Grant } from '@dgrants/types';
+import { GrantEthers } from '@dgrants/types';
```

Comment:
why are using GrantEthers ? not super clear on this change

```
export type GrantEthers = GrantArray & GrantObject;
export type Grant = GrantObject | GrantEthers;
```

Reply:
Because `Grant` (that we use internally) now uses `id: number`, but we still need to _type_ the same data when we pull it from chain before we convert `GrantEthers` to `Grant`